lib/utils/cli.go

```diff
@@ -178,21 +178,15 @@ func UserMessageFromError(err error) string {
 // The error message is escaped if necessary. A newline is added if the error text
 // does not end with a newline.
 func FormatErrorWithNewline(err error) string {
```

Comment:
Should we capture the changes in a simple test? Do you think that's worth it?

Reply:
Fixed an existing test